packages/docs/src/components/doc/Playground.svelte

```diff
@@ -93,7 +93,15 @@
               <span class="text--secondary">{label}</span>
             </Switch>
           {:else if controls[label].type === 'slider'}
-            <Slider bind:value={values[label]}>{label}</Slider>
+            <Slider
+              bind:value={values[label]}
+              min={controls[label].min || 0}
```

Comment:
these newly added settings duplicate the defaults of `Slider` which feels wrong as they might diverge.

Reply:
works for now, or what would you do instead? 😄